src/api/common/hybrid/index.js

```diff
@@ -1,36 +1,43 @@
 import testRunTracker from './test-run-tracker';
 import compiledCode from './compiled-code-symbol';
+import replicator from './replicator';
 import TestRun from '../../../test-run';
-import compileHybridFunction from '../../../compiler/es-next/compile-hybrid-function';
-import { ExecuteClientCodeCommand } from '../../../test-run/commands';
+import { compileHybridFunction } from '../../../compiler/es-next/hybrid-function';
+import { ExecuteHybridFunctionCommand } from '../../../test-run/commands';
```

Comment:
@inikulin You have changed `ExecuteHybridFunctionCommand` to `ExecuteClientCodeCommand` before, now you revert it. Could you please explain what the reason of this and what naming we will use filnally?